Gems/OpenXRVk/Code/Source/OpenXRVkInput.cpp

```diff
@@ -356,7 +356,26 @@ namespace OpenXRVk
             LocateVisualizedSpace(device->GetPredictedDisplayTime(), session->GetXrSpace(spaceType),
                                   session->GetXrSpace(OpenXRVk::SpaceType::View), spaceType);
         }
+        
+        // XR to AZ vector conversion...
```

Comment:
In the comment add the fact that it is going from y-up to z-up

Reply:
Will do.